MSAL/test/automation/tests/interactive/MSALAADBasicInteractiveTests.m

```diff
@@ -382,7 +382,7 @@ - (void)testCompanyPortalInstallPrompt_withNonConvergedApp_withSystemWebView
     [self acceptAuthSessionDialog];
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->